mesonbuild/scripts/depfixer.py

```diff
@@ -364,38 +364,49 @@ def get_darwin_rpaths_to_remove(fname):
             result.append(rp)
     return result
 
-def fix_darwin(fname, new_rpath):
+def fix_darwin(fname, new_rpath, final_path, install_name_mappings):
     try:
         rpaths = get_darwin_rpaths_to_remove(fname)
     except subprocess.CalledProcessError:
         # Otool failed, which happens when invoked on a
         # non-executable target. Just return.
         return
     try:
+        args = []
         if rpaths:
-            args = []
             for rp in rpaths:
                 args += ['-delete_rpath', rp]
             subprocess.check_call(['install_name_tool', fname] + args,
                                   stdout=subprocess.DEVNULL,
                                   stderr=subprocess.DEVNULL)
+        args = []
         if new_rpath:
-            subprocess.check_call(['install_name_tool', '-add_rpath', new_rpath, fname],
+            args += ['-add_rpath', new_rpath]
+        # Rewrite -install_name @rpath/libfoo.dylib to /path/to/libfoo.dylib
```

Comment:
are you using absolute `install_name`s now instead of relative ones? This would be a major breaking change

Reply:
> are you using absolute install_names now instead of relative ones?

Yes, see commit message.

> This would be a major breaking change

Can't break what is already broken ;)